api/envoy/extensions/tracers/opentelemetry/samplers/v3/cel_sampler.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.extensions.tracers.opentelemetry.samplers.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.tracers.opentelemetry.samplers.v3";
+option java_outer_classname = "CelSamplerProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/tracers/opentelemetry/samplers/v3;samplersv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Always On Sampler config]
+// Configuration for the "CEL" Sampler extension.
+//
+// [#extension: envoy.tracers.opentelemetry.samplers.cel]
+
+message CELSamplerConfig {
+  // Expression that, when evaluated, will be used to make sample decision.
+  string expression = 1;
+}
```

Comment:
What will happen if empty string is configured?

Reply:
add tests for it.

Reply:
@zirain this would be helpful to document the behavior for the empty string. Also if it has to be non-empty please add a constraint.

Reply:
update validate rule.